images/miq-app/docker-assets/container-scripts/container-deploy-common.sh

```diff
@@ -5,18 +5,21 @@
 # This file is created by the write_deployment_info during initial deployment
 PV_DEPLOY_INFO_FILE="${APP_ROOT_PERSISTENT}/.deployment_info"
 
-# This directory is used to store application data to be persisted
+# This directory is used to store server specific data to be persisted
 PV_CONTAINER_DATA_DIR="${APP_ROOT_PERSISTENT}/container-data"
 
-# This directory is used to store container deployment data (logs,backups,etc)
+# This directory is used to store server specific container deployment data (logs,backups,etc)
 PV_CONTAINER_DEPLOY_DIR="${APP_ROOT_PERSISTENT}/container-deploy"
 
-# This directory is used to store initialization logfiles on PV
+# This directory is used to store server specific initialization logfiles on PV
 PV_LOG_DIR="${PV_CONTAINER_DEPLOY_DIR}/log"
 
-# Directory used to backup PV data before performing an upgrade
+# Directory used to backup server specific PV data before performing an upgrade
 PV_BACKUP_DIR="${PV_CONTAINER_DEPLOY_DIR}/backup"
 
+# This directory is used to store shared region application data to be persisted (database.yml, keys, etc)
+PV_CONTAINER_DATA_REGION_DIR="${APP_ROOT_PERSISTENT_REGION}/container-data"
```

Comment:
Yes, I've been going back and forth with the location of deployment-info, let's discuss.